target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -262,10 +254,16 @@
 
 	sram0: sram@9f000000 {
 		compatible = "mmio-sram";
-		reg = <0x9f000000 0x10000>;
+		reg = <0x9f000000 0x20000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		ranges = <0 0x9f000000 0x10000>;
+		ranges = <0 0x9f000000 0x20000>;
```

Comment:
Separate commit with Fixes: tag? Then that can be merged ahead of some of the bigger changes.